celery/result.py

```diff
@@ -823,8 +824,12 @@ class GroupResult(ResultSet):
     #: List/iterator of results in the group
     results = None
 
-    def __init__(self, id=None, results=None, **kwargs):
+    #: Parent Result of the group, if any
+    parent = None
```

Comment:
Yes, it seems to be a class variable in `ResultBase`, so I would keep 818, but remove 828 as it is redundant.